oauthdoctor/diag/config.go

```diff
@@ -48,12 +48,14 @@ const (
 	RefreshToken = "RefreshToken"
 )
 
-// PIIWords is a slice of constant strings that indicate Personally Identifiable Information
-var PIIWords = []string{DevToken, ClientID, ClientSecret, RefreshToken}
+var (
+	// PIIWords is a slice of constant strings that indicate Personally Identifiable Information
+	PIIWords = []string{DevToken, ClientID, ClientSecret, RefreshToken}
 
-// RequiredKeys are the key names used in the Language structure that defines
-// the contents of a client library configuration file.
-var RequiredKeys = []string{DevToken, ClientID, ClientSecret, RefreshToken}
+	// RequiredKeys are the key names used in the Language structure that defines
+	// the contents of a client library configuration file.
+	RequiredKeys = []string{DevToken, ClientID, ClientSecret, RefreshToken}
```

Comment:
If these contain the same constants, do we need two slices just with different names.

Reply:
The elements in the slices just happen to be the same. They can diverge at some point when more config keys are added.